README.md

```diff
@@ -40,10 +49,10 @@ Stamps have a method called `.compose()`:
 stamp.compose(...args?: stamp|descriptor[]): stamp
 ```
 
-When called the `.compose()` method creates new composable using the current composable as a base, composed with a list of *composables* or *descriptors* passed as arguments:
+When called the `.compose()` method creates new stamp using the current stamp as a base, composed with a list of *composables* or *descriptors* passed as arguments:
```

Comment:
I didn't decide it on my own, it just wasn't explicitly clear in the readme. Remember that everybody had a chance to weigh in on the example code which clarified the terminology in working code.